packages/fxa-content-server/app/scripts/templates/sign_up_password.mustache

```diff
@@ -35,16 +35,16 @@
       {{{ coppaHTML }}}
 
       {{^isCWTSOnSignupPasswordEnabled}}
-      <section class="get-involved">
+      <section class="mb-5">
       {{#isAnyNewsletterEnabled}}
           <header>
-              <h2>{{#t}}Practical knowledge is coming to your inbox. Sign up for more:{{/t}}</h2>
+              <h2 class="mb-5 text-base ltr:text-left rtl:text-right">{{#t}}Practical knowledge is coming to your inbox. Sign up for more:{{/t}}</h2>
```

Comment:
Re: `ltr:text-left rtl:text-right`, what about usin' `text-start`?
- https://tailwindcss.com/docs/text-align
- https://developer.mozilla.org/docs/Web/CSS/text-align

> [**Values**](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#values)
> - `start` &mdash; The same as `left` if direction is left-to-right and `right` if direction is right-to-left.
> - `end` &mdash; The same as `right` if direction is left-to-right and `left` if direction is right-to-left.

Reply:
Ooo I'm not sure if I even knew that was a possible value for `text-align` which is crazy to me, seems like we should use it everywhere. I'm tempted to change them all in this PR since there's only two components in Settings using that and the others are related to changes I've made in the last few weeks.